tests/snapshots/test_es_datastream_manager.ambr

```diff
@@ -43,22 +43,6 @@
             }),
             'agent': dict({
               'properties': dict({
-                'ephemeral_id': dict({
-                  'ignore_above': 1024,
-                  'type': 'keyword',
-                }),
-                'id': dict({
-                  'ignore_above': 1024,
-                  'type': 'keyword',
-                }),
-                'name': dict({
-                  'ignore_above': 1024,
-                  'type': 'keyword',
-                }),
-                'type': dict({
-                  'ignore_above': 1024,
-                  'type': 'keyword',
-                }),
```

Comment:
What's the reason for these removed fields?

Reply:
They are unused, the only agent field we populate is agent.version with the hass version